### DISCUSSION TITLE
How to use NuGetAdd on a specific project, not the whole solution

### DISCUSSION THREAD

**Discussion by DWIAltonaAnalytics** [cake-build/cake#4186] [Extension Q&A] [answered]
As far as I understand the documentation, there is no way to use `NuGetAdd` and tell it where to install the NuGet package. I have some test projects, where I would like to have a NuGet-Package present when a parameter was set, and not otherwise.
Am I missing an alias? Because for example `DotNetBuild` takes an argument to tell it *what* it is supposed to build.

**Comment by augustoproiete** [comment, marked as answer]
@DWIAltonaAnalytics `NuGetAdd` behind the scenes calls `nuget.exe add ...` - Cake is only providing C# a syntax for a command-line utility - https://learn.microsoft.com/en-us/nuget/reference/cli-reference/cli-ref-add

**Reply by DWIAltonaAnalytics** [reply]
so there should also be DotnetAddPackage alias, which would call `dotnet add [<Project>] package <PackageName>` behind the scenes, so we could add NuGets to specific projects.

**Reply by pascalberger** [reply]
In the meantime you can always use a [Command](https://cakebuild.net/dsl/command/) or [Process](https://cakebuild.net/dsl/process/) alias to make any call for which no alias exists.
